README.md

```diff
@@ -158,6 +158,99 @@ The following environment parameters are available:
 
 If you use `SWISNAP_ENABLE_<plugin_name>` set to `true`, then keep in mind that AppOptics Host Agent will use default plugins configs and task manifests. For custom configuration see [Custom plugins configuration and tasks manifests](###custom-plugins-configuration-and-tasks-manifests).
 
+## Events collector
+
+Version 22 of Kubernetes collector allows you to collect cluster events and , together with logs collector push them to Loggly. To enable event collection in your deployment, follow these easy steps:
+* Create `kubernetes.yaml` file, that will configure kubernetes collector. This config should contains `collector.kubernetes.all.events` field with specified filter. Following example config will watch for normal events in default namespace:
+  ```yaml
+  collector:
+    kubernetes:
+      all:
+        incluster: true
+        kubeconfigpath: ""
+        interval: "60s"
+
+        events: |
+          # Embedded YAML (as a multiline string literal)
+          filters:
+          - namespace: default
+            type: normal
+
+        grpc_timeout: 30
+
+  load:
+    plugin: snap-plugin-collector-aokubernetes
+    task: task-aokubernetes.yaml
+  ```
+* If you want to monitor events count in AppOptics, then edit your current `task-aokubernetes.yaml` task manifest so it contains `/kubernetes/events/count` metric in `workflow.collect.metrics` list, and copy it to working directory:
+  ```yaml
```

Comment:
maybe a diff would be more readable here?

Reply:
not really, if I would run diff against default task-manifest I'd get lots of deleted  comments and stuff